src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1729,20 +1731,31 @@ public static bool TryConvertTo(object valueToConvert, Type resultType, out obje
         /// <returns>false for conversion failure, true for success</returns>
         public static bool TryConvertTo(object valueToConvert, Type resultType, IFormatProvider formatProvider, out object result)
         {
-            result = null;
+            result = default;
             try
             {
-                result = ConvertTo(valueToConvert, resultType, formatProvider);
+                using (typeConversion.TraceScope("Converting \"{0}\" to \"{1}\".", valueToConvert, resultType))
+                {
+                    if (resultType == null)
+                    {
+                        return false;
+                    }
+
+                    var conversion = FigureConversion(valueToConvert, resultType, out var debase);
+                    if (conversion.Rank== ConversionRank.None)
```

Comment:
Please add space before `==`.

Reply:
Please address the comment and remove extra space 
`conversion.Rank== ConversionRank.None` -> `conversion.Rank == ConversionRank.None`